scripts/west_commands/runners/nrfjprog.py

```diff
@@ -76,16 +81,24 @@ def do_add_parser(cls, parser):
         parser.add_argument('--force', required=False,
                             action='store_true',
                             help='Flash even if the result cannot be guaranteed.')
+        parser.add_argument('--recover', required=False,
+                            action='store_true',
+                            help='''erase all user available non-volatile
```

Comment:
We should mention that this erases both cores on the nRF53, regardless of which core you are actually flashing after